setup/config/config_clientapi.go

```diff
@@ -88,7 +88,7 @@ func (c *ClientAPI) Verify(configErrs *ConfigErrors, isMonolith bool) {
 			c.RecaptchaFormField = "g-recaptcha"
```

Comment:
This should be `g-recaptcha-response` instead - it's what recaptcha sends to the server upon a successful form submission.

```suggestion
			c.RecaptchaFormField = "g-recaptcha-response"
```